addons/website/static/tests/tours/page_manager.js

```diff
@@ -29,6 +29,27 @@ const checkKanbanGroupBy = [{
     trigger: '.o_cp_searchview .o_facet_remove',
 }];
 
+const checkWebsiteFilter = [
+    {
+        content: "Click on My Website",
+        trigger: "button.dropdown-toggle:contains('My Website')",
+    }, {
+        content: "Select My Website 2",
+        trigger: ".dropdown-menu.show > .dropdown-item:contains('My Website 2')",
+    }, {
+        content: "Check that the homepage is now the one of My Website 2",
+        trigger: ".o_list_table .o_data_row .o_data_cell[name=name]:contains('Home') " +
+                "~ .o_data_cell[name=website_id]:contains('My Website 2')",
```

Comment:
Not a big deal and linter wise ok but looks weird
```suggestion
        trigger: ".o_list_table .o_data_row .o_data_cell[name=name]:contains('Home') " +
                 "~ .o_data_cell[name=website_id]:contains('My Website 2')",
```